crates/node/src/service.rs

```diff
@@ -64,6 +64,32 @@ type FullSelectChain = sc_consensus::LongestChain<FullBackend, Block>;
 type BasicImportQueue<Client> = sc_consensus::DefaultImportQueue<Block, Client>;
 type BoxBlockImport<Client> = sc_consensus::BoxBlockImport<Block, TransactionFor<Client, Block>>;
 
+
+/// Creates a new set of partial components for the Madara blockchain.
+///
+/// Constructs and initializes various components needed for the Madara blockchain
+/// using the provided configuration and builder functions.
+///
+/// # Arguments
+///
+/// - `config`: A reference to the Madara blockchain's configuration.
+/// - `build_import_queue`: A function that builds the import queue and block import for the
+///                          blockchain. It takes various parameters and returns a result
+///                          containing the import queue and block import components, or an
+///                          error if the construction fails.
+///
+/// # Generic Parameters
+///
+/// - `RuntimeApi`: The runtime API trait that should be used with the blockchain. It should be
+///                  a type that implements `ConstructRuntimeApi` trait for the `Block` and
+///                  `FullClient` types.
```

Comment:
`It should be a type that implements `ConstructRuntimeApi` trait for the `Block` and FullClient` types.`
 This is already said by the code. You can remove it